pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -284,6 +313,38 @@ func (ctrl *Controller) itmsConfDeleted(_ interface{}) {
 	ctrl.imgQueue.Add("openshift-config")
 }
 
+func (ctrl *Controller) addImagePolicyObservers() {
+	ctrl.configInformerFactory.Config().V1alpha1().ClusterImagePolicies().Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    ctrl.clusterImagePolicyAdded,
+		UpdateFunc: ctrl.clusterImagePolicyUpdated,
+		DeleteFunc: ctrl.clusterImagePolicyDeleted,
+	})
+	ctrl.clusterImagePolicyLister = ctrl.configInformerFactory.Config().V1alpha1().ClusterImagePolicies().Lister()
+	ctrl.clusterImagePolicyListerSynced = ctrl.configInformerFactory.Config().V1alpha1().ClusterImagePolicies().Informer().HasSynced
+}
+
+func (ctrl *Controller) clusterImagePolicyAdded(_ interface{}) {
+	ctrl.imgQueue.Add("openshift-config")
+}
+
+func (ctrl *Controller) clusterImagePolicyUpdated(_, _ interface{}) {
+	ctrl.imgQueue.Add("openshift-config")
+}
+
+func (ctrl *Controller) clusterImagePolicyDeleted(_ interface{}) {
+	ctrl.imgQueue.Add("openshift-config")
+}
+
+func (ctrl *Controller) sigstoreAPIEnabled() bool {
+	featureGates, err := ctrl.featureGateAccess.CurrentFeatureGates()
+	if err != nil {
+		klog.Infof("error getting current featuregates: %v", err)
+		return false
+	}
+	klog.Infof("getting sigstoreImageVerification featureGate value: %v", featureGates)
```

Comment:
(Is it necessary to log this on every sync iteration? Quite likely it doesn’t matter, it’s just one more log line — I have no idea whether that’s material.)

Reply:
oh..typo, just want to log the sigstoreImageVerification value here

Reply:
Oh… that’s a good change I didn’t notice was necessary :) 

---

My point here was that `sigstoreAPIEnabled` is called (and logs this) on every `syncImageConfig`. I don’t think that’s really _necessary_ — the `addded image policy observers with sigstore featuregate enabled` log in `Run` records the state — but I also didn’t check whether a single run of `syncImageConfig` produces 0/1/10/100 Info-level log lines, so I don’t know whether it matters at all.